### PR TITLE
feat: Improve and add new features to outliers API

### PR DESCRIPTION
**Pull request by d-bernat**

### CODE REVIEW

**Review by maikelarabori — CHANGES_REQUESTED** [review]
Looks good. Two minor changes are required. 
Thanks!

**Comment by maikelarabori** [comment]
@d-bernat, please start adding descriptions to your PRs, so we know what's being implemented and the motivation of the PR. It's also important because it might help for future investigations/references...

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/outlier/OrderDirection.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package org.hisp.dhis.analytics.outlier;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
```
[the remark this answers is not in the document]
Comment (d-bernat) [reply dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/outlier/OrderDirection.java]: done

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/outlier/data/OutlierQueryParams.java [hunk 2]
```diff
@@ -67,6 +68,8 @@ public class OutlierQueryParams {
 
   private Order orderBy;
 
+  private OrderDirection orderDirection;
```
[the remark this answers is not in the document]
Comment (d-bernat) [reply dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/outlier/data/OutlierQueryParams.java]: done

**Review by maikelarabori — APPROVED** [review]

**Review by gnespolino — APPROVED** [review]

**Pull request merged**